transaction-view/src/lib.rs

```diff
@@ -4,16 +4,12 @@ pub mod bytes;
 #[cfg(not(feature = "dev-context-only-utils"))]
 mod bytes;
 
-#[allow(dead_code)]
-pub mod address_table_lookup_meta;
-#[allow(dead_code)]
+mod address_table_lookup_meta;
 mod instructions_meta;
-#[allow(dead_code)]
 mod message_header_meta;
 pub mod result;
-#[allow(dead_code)]
 mod signature_meta;
-#[allow(dead_code)]
 pub mod static_account_keys_meta;
```

Comment:
this was stays public since a constant from it is used.
the internal struct is pub(crate) so not exposed publicly